scripts/remap_sections.py

```
@@ -0,0 +1,1271 @@
+
+    Produces the "what would be restored on real dev" report without writing
+    anything. ``client`` must expose ``list_receipt_lines_from_receipt``.
+    """
+    stats = Stats()
+    per_receipt: list[dict[str, Any]] = []
+    for idx, packet in enumerate(packets, start=1):
+        image_id = packet["image_id"]
+        receipt_id = int(packet["receipt_id"])
+        try:
+            new_lines = read_new_lines(client, image_id, receipt_id)
+        except Exception as exc:  # pragma: no cover - live path
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)